### DISCUSSION TITLE
What is your favorite type of game???

### DISCUSSION THREAD

**Discussion by Kudo-Takuto** [NAIST-SE/AttractingContributors#928] [2023 Attraction Contributions]
I like games of all kinds.
And I would like to know your favorite games!

So please come to my fork and tell me!:
https://github.com/Kudo-Takuto/AttractingContributors.git

**Comment by bconfessor** [comment]
Done! Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md

**Comment by ppjaisri** [comment]
I love Minecraft
I sent you a pull requests
Would you like to make Minecraft totally epic ?
https://github.com/ppjaisri/AttractingContributors-Install-add-on-to-Minecraft
